andela/prometheus-ah-frontend
# #159987670:Authenticated Users can like/unlike articles

#### What does this PR do?
- This PR is to give Authenticated users the ability to like and unlike an article
#### Description of Task to be completed?
- Write code to implement the functionality of the feature
- Write tests for the actions, reducers and components to check against the functionality of the code
#### How should this be manually tested?
- When the application is up and running
- Navigate to the read article page
- Click on a like icon to like the article
- You should see the icon change depending on your like status for the article
#### What are the relevant pivotal tracker stories?(if applicable)
[#159987670](https://www.pivotaltracker.com/story/show/159987670)
